packages/node/src/breadcrumbs/FileBreadcrumbsStorage.ts

```diff
@@ -0,0 +1,77 @@
+import {
+    BacktraceAttachment,
+    Breadcrumb,
+    BreadcrumbLogLevel,
+    BreadcrumbType,
+    BreadcrumbsStorage,
+    RawBreadcrumb,
+    SessionFiles,
+    TimeHelper,
+    jsonEscaper,
+} from '@backtrace-labs/sdk-core';
+import path from 'path';
+import { BacktraceFileAttachment } from '../attachment';
+import { AlternatingFileWriter } from '../common/AlternatingFileWriter';
+
+const FILE_PREFIX = 'breadcrumbs';
+
+export class FileBreadcrumbsStorage implements BreadcrumbsStorage {
+    public get lastBreadcrumbId(): number {
+        return this._lastBreadcrumbId;
+    }
+
+    private _lastBreadcrumbId: number = TimeHelper.toTimestampInSec(TimeHelper.now());
+
+    private readonly _writer: AlternatingFileWriter;
+
+    constructor(private readonly _file1: string, private readonly _file2: string, maximumBreadcrumbs: number) {
```

Comment:
how about renaming them to:
_file1 => _sourceFile
_file2 => _backupFile

Reply:
Changed to `_mainFile` and `_fallbackFile`